osgp/platform/osgp-adapter-kafka-distributionautomation/pom.xml

```diff
@@ -11,19 +11,21 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
+  <artifactId>osgp-adapter-kafka-distributionautomation</artifactId>
```

Comment:
Undo this reordering. See also sonar rule "_pom elements should be in the recommended order (java:S3423)_":

_The POM Code Convention is the Maven project's internal recommendation for POM element ordering. It calls for listing modifiers in the following order:_

```
<modelVersion/>
<parent/>
<groupId/>
<artifactId/>
<version/>
<packaging/>
<name/>
<description/>
<url/>
<inceptionYear/>
<organization/>
<licenses/>
<developers/>
<contributors/>
<mailingLists/>
<prerequisites/>
<modules/>
<scm/>
<issueManagement/>
<ciManagement/>
<distributionManagement/>
<properties/>
<dependencyManagement/>
<dependencies/>
<repositories/>
<pluginRepositories/>
<build/>
<reporting/>
<profiles/>
```

Reply:
Hmm, I  just made this in line with all other modules. I'll revert this change